optaplanner-docs/src/main/asciidoc/ConstraintStreams/ConstraintStreams-chapter.adoc

```diff
@@ -176,6 +176,31 @@ The following constraint stream cardinalities are currently supported:
 |4          |  Quad|`QuadConstraintStream<A, B, C, D>`
 |===
 
+[[constraintStreamsHigherCardinalities]]
+=== Achieving higher cardinalities
+
+OptaPlanner currently does not support constraint stream cardinalities higher than 4.
+However, with <<constraintStreamsMappingTuples,tuple mapping>> it is easy to achieve effectively infinite cardinality:
```

Comment:
docs and blogs should avoid "it is easy" (although I suspect more docs chapter I wrote suffer from this).

"it is easy" is subjective - it depends on previous experience and understanding of context.
Worse, it also implies an intelligence test for the reader. -> If you can't understand this quickly, you're an idiot. => OptaPlanner is too difficult for me, I need to look for a simpler solution.